src/app/project-links.ts

```diff
@@ -4,16 +4,16 @@ import { Injectable } from "@angular/core";
 export default class ProjectLinks {
     public static urlHome = "/";
     public static urlProfile = "profile";
-    public static urlLogin = "login";
+    public static urlLogin = "v/login";
     public static urlGithubCallback = "github_callback";
-    public static urlSearch = "search";
-    public static urlDatasetView = "dataset-view";
+    public static urlSearch = "v/search";
+    //  public static urlDatasetView = "dataset-view";
```

Comment:
If you no longer need some of these constants, just kill them.
Note, 2 constants at the end, `urlDatasetCreateSelectType` and `urlDatasetCreateRoot`, were probably reserved by Yulia for not yet completed branches.

Reply:
Ok, I'd better leave the last 2 costants